### PR TITLE
adapt to new xgboost function call

### PR DESCRIPTION
**Pull request by ivokwee**
Seems function call has changed for xgboost. Now gives error.

1. Parameters data and labels are now x and y. Using non-named paramters.
2. eta is deprecated. now learning_rate. Just remove and use default
3. multi:softmax not available, using multi:softprob
4. For label (y) to be character instead of previous integer
5. remove (deprecated) verbose parameter

NB: should still be compatible with older version call

### CODE REVIEW

**Review by zitoa — APPROVED** [review]

**Comment by ESCRI11** [comment]
Ok new images will have latest xgboost version, approving

**Review by ESCRI11 — APPROVED** [review]
